ui-tests/src/test/java/org/openmrs/reference/FormTest.java

```diff
@@ -1,5 +1,7 @@
 package org.openmrs.reference;
 
+import static org.junit.Assert.assertNotNull;
```

Comment:
Organizing the order of imports to follow the OpenMRS `imports convention` is a good idea to ensure quality within our code base. @sherrif10 cross check with the existing files to see the import order.

Reply:
Hello @kdaud , I  think this is the right way imports should be organised

Reply:
Uhmm But when i save still this import remains on top , i dont understand by saying following the alphabetic order, imports should be well formatted and organised by the ide. i dont see any problem with that  cc @ibacher

Reply:
> Uhmm But when i save still this import remains on top , i dont understand by saying following the alphabetic order, imports should be well formatted and organised by the ide

Some configuration within the IDE results into this! However, this can be handled with ease when the IDE is well configured!

>  i dont see any problem with that 

BTW this is not a bug and has no harm at all with other existing code base. However, this means we are not ensuring standards within our code base with is not a good idea

Reply:
My ide is well configured and still i think this is the right way,i actually dont mind about the right way however my ide is well configured to automatically organise import statements, i can say the first commit which was merged that first authores same imports where the later was first also had no issus. it doesnot matter which is first which is last, what matters is whether your imports are well organised. what i mean is 

There is a difference btn  
i**mport org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.Select;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.net.MalformedURLException;
import java.net.URL;
import java.time.Duration;**

And 
**import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.Select;
import org.openqa.selenium.support.ui.WebDriverWait;
import java.net.MalformedURLException;
import java.net.URL;
import java.time.Duration;**

And there is no difference btn

import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.Select;
import org.openqa.selenium.support.ui.WebDriverWait;
import java.net.MalformedURLException;
import java.net.URL;
import java.time.Duration;

and 
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.Select;
import org.openqa.selenium.support.ui.WebDriverWait;

**import java.net.MalformedURLException;
import java.net.URL;
import java.time.Duration;**

The only difference is that the second last is not well organised which is not allowed as openmrs conventions , So according to your suggestions as if the first should come last which is not a strong decision to lie on , And if i would have time i would have shown you different code that have some thing similar.  cc @ibacher  whats your suggestion

Reply:
Static imports (like this one) always come before standard imports (note `import static` rather than just `import`), so this is actually inline with our coding standards.